src/mavesp8266_parameters.h

```diff
@@ -41,12 +41,16 @@
 #define WIFI_MODE_AP 0
 #define WIFI_MODE_STA 1
 
+#define CAST_MODE_UNI 0
+#define CAST_MODE_MULTI 1
```

Comment:
Should we really change the default?

Reply:
As discussed on discord, we are not changing any defaults here. Thanks.